dimensional.cabal

```diff
@@ -62,7 +63,8 @@ library
                        Numeric.Units.Dimensional.UnitNames,
                        Numeric.Units.Dimensional.UnitNames.InterchangeNames,
                        Numeric.Units.Dimensional.Variants
-  other-modules:       Numeric.Units.Dimensional.UnitNames.Internal
+  other-modules:       Numeric.Units.Dimensional.Internal,
```

Comment:
AIUI you're correct that that's the difference between `exposed-modules:` and `other-modules:`. Such an import will fail, and I did intend this. It preserves the status quo, because the export list of `Numeric.Units.Dimensional` didn't change at all, only some parts of it moved to `.Internal`. The reason for this is to expose those internals to other modules within the project. For example, the `.Coercion` module imports the newtype constructor `Quantity` from there and reexports it for users who want to promise that they know what they're doing, they can then use the `Data.Coerce` machinery once they have it in scope. But my larger reason for changing this was because it lays the groundwork for my work-in-progress `.FixedPoint` module which needs access to these internals in the same way that `Numeric.Units.Dimensional` does in order to define fixed-point analogs of the arithmetic operations that are defined there.